src/main/java/uk/gov/companieshouse/disqualifiedofficersdataapi/service/DisqualifiedOfficerService.java

```diff
@@ -110,15 +110,17 @@ public void deleteDisqualification(String contextId, String officerId) {
      */
     private void deleteCorporateDisqualification(String contextId, String officerId) {
         CorporateDisqualificationDocument document = retrieveCorporateDisqualification(officerId);
-        repository.delete(document);
-        logger.info(String.format("Corporate disqualification is deleted in MongoDb officer id: %s",
-            contextId,
-            officerId));
+
         document.getData().setKind(KindEnum.CORPORATE_DISQUALIFICATION);
         disqualifiedOfficerApiService.invokeChsKafkaApi(
-                new ResourceChangedRequest(contextId, officerId, 
+                new ResourceChangedRequest(contextId, officerId,
                         DisqualificationResourceType.CORPORATE, document.getData(), true));
         logger.info(String.format("ChsKafka api DELETED invoked updated successfully for context id: %s and officer id: %s",
+                contextId,
+                officerId));
+
+        repository.delete(document);
+        logger.info(String.format("Corporate disqualification is deleted in MongoDb officer id: %s",
```

Comment:
Officer ID is being incorrectly logged as context ID here

Reply:
Probably a typo, have made it it equivalent to the Natural disqualifications delete logging message on line 146.